edge-server/anthos/bmctl-physical-template.yaml

```diff
@@ -59,7 +59,8 @@ spec:
   clusterOperations:
     projectID: ${GOOGLE_CLOUD_DEFAULT_PROJECT}
     location: ${GOOGLE_CLOUD_DEFAULT_REGION}
-    enableApplication: true
+    # https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/how-to/application-logging-monitoring
+    # enableApplication: true
```

Comment:
Why do we keep this?

Reply:
was intended to leave here as a comment, removed